File: 백준/Silver/11399. ATM/ATM.js
const fs = require("fs");

const [n, rest] = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

const arr = rest.split(" ").map(Number).sort((a, b) => {
    return a - b;
  });

let result = 0;
let sum = 0;

for (e of arr) {
  sum += e;
  result += sum;
}

console.log(result);

File: 백준/Silver/11399. ATM/ATM.cc
using namespace std;
#include <iostream>
#include <algorithm>
#include <vector>

int main()
{
    int N;
    cin >> N;
    
    vector<int> v(N);
    for (int i = 0; i < N; ++i) cin >> v[i];
    sort(v.begin(), v.end());
    
    int result(0), sum(0);
    for (int num : v)
    {
        sum += num;
        result += sum;
    }
    
    cout << result;
}